rust/hyperlane-base/src/contract_sync/metrics.rs

```diff
@@ -1,25 +1,26 @@
 use crate::CoreMetrics;
 use prometheus::{IntCounterVec, IntGaugeVec};
-use std::sync::Arc;
 
 /// Struct encapsulating prometheus metrics used by the ContractSync.
 #[derive(Debug, Clone)]
 pub struct ContractSyncMetrics {
+    // TODO: Does this still apply?
```

Comment:
Curious for thoughts on what metrics we should keep here? The forwards/backwards style cursor maybe makes the previous idea of a single "indexed height" and "message nonce" per event type not applicable

Reply:
We could just have the current cursor values

Reply:
Sure but the ForwardBackward cursor jumps all over the place.

We *could* put the metrics in the cursor, and have it handle it, and the forward/backward cursor could use different labels

Reply:
That would make sense to me

Reply:
I updated it to monotonically increase instead. So we will just not track the "backwards" part of the forward/backward sync, for now